### ISSUE TITLE
doubt about the loss computation in the train.py

### ISSUE THREAD

**Issue by ZhouHaoWa** [zoubohao/DenoisingDiffusionProbabilityModel-ddpm-#10]
loss = trainer(x_0).sum() / 1000.
why is 1000?

**Comment by HBB0517** [comment]
because 1000 can create a isotropic

**Comment by huchenz1** [comment]
> 

could you pleasr tell me what does "isotropic" mean? Thanks!

**Comment by huchenz1** [comment]
> because 1000 can create a isotropic


could you pleasr tell me what does "isotropic" mean? Thanks!

**Comment by xpdd123** [comment]
just make gradient desent faster

**Comment by Klawens** [comment]
It's common to scale the loss by a constant factor to control the magnitude of gradients during training. Large gradients can lead to unstable training, so scaling the loss down can help stabilize the optimization process.
